src/test/java/LoginTests.java

```diff
@@ -8,35 +9,37 @@
 public class LoginTests extends BaseTest {
 
     //    @Test(enabled = true, priority = 0, description = "LoginEmptyEmailPasswordTest")
-    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
-    public static void loginEmptyEmailPasswordTest (String email, String password) {
-
-        login(email, password);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
+//    @Test(dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
+//    public static void loginEmptyEmailPasswordTest (String email, String password) {
+//        //login(email, password);
+//        LoginPage loginPage = new LoginPage(getDriver());
+//        loginPage.provideEmail("demo@class.com");
+//        loginPage.providePassword("");
+//        loginPage.clickSubmitBtn();
+//        //wait.
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
 
     //Page Object Model example
     @Test
     public void LoginValidEmailPasswordTest () {
 
-        LoginPage loginPage = new LoginPage(driver);
-        HomePage homePage = new HomePage(driver);
-
+        //LoginPage loginPage = new LoginPage(driver);
+        //HomePage homePage = new HomePage(driver);
+         LoginPage loginPage = new LoginPage(getDriver());
+         HomePage homePage = new HomePage(getDriver());
         loginPage.provideEmail("demo@class.com");
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)